types/default-theme.d.ts

```diff
@@ -393,13 +396,34 @@ export namespace DefaultTheme {
        * @see https://lucaong.github.io/minisearch/modules/_minisearch_.html#searchoptions-1
        */
       searchOptions?: MiniSearchOptions['searchOptions']
-    }
 
+      /**
+       * Overrides the default regex based page splitter.
+       * Supports async generator, making it possible to run in true parallel
+       * (when used along with `node:child_process` or `worker_threads`)
+       * ---
+       * This should be especially useful for scalability reasons.
+       * ---
+       * @param {string} path - absolute path to the markdown source file
+       * @param {string} html - document page rendered as html
+       */
+      splitIntoSections?: (
```

Comment:
prefix this with underscore, otherwise it will be included in client side code

Reply:
done